server/server.js

```diff
@@ -0,0 +1,90 @@
+var express = require('express');
+var app = express();
+var port = 3000;
+var cors = require('cors')
+var task = require("./task");
+app.use(cors());
+// app.use(express.static(__dirname + '../html'));
+// app.use(express.static(__dirname + '../js'));
+// app.use(express.static(__dirname + '../node_modules'));
+
+
+//express
+var bodyParser = require("body-parser");
+app.use(bodyParser.json());
+
+app.post("/todo", function(req, res,next) {
+          task.create({"title": req.body.title,"dueDate": req.body.dueDate,"address": req.body.address,"lat": req.body.lat,"lng": req.body.lng}, function(err, task) {
+            if (err) {
+              res.status(500);
+              res.send(err);
+            } else {
+              res.status(201);
+              res.send(task);
+            }
+          });
+})
+
+app.delete("/todo/:id", function(req, res,next) {
+  var id = '' + req.params.id;
+  console.log("here");
+  task.remove({
+    "_id": id
+  }, function(err, result) {
+    if (err) {
+      res.status(500);
+
+      next("Internal server error.");
+    } else if (result.n == 0) {
```

Comment:
Is result.n the number of results? That's good to know!

Reply:
Yeah it is. There is also one unnecessary space isn't i :-P